i18n/zh/docusaurus-plugin-content-docs-community/current/4-committer.md

```diff
@@ -17,6 +17,18 @@ cover: "/img/architecture/shenyu-framework.png"
 * Apache ShenYu仓库写的权限
 * [jetbrains正版全家桶](https://www.jetbrains.com/shop/eform/apache)
 
+## 提名流程
+
+**1、按照这个 [Committer 指南](https://community.apache.org/newcommitter.html) 完成投票**
+
```

Comment:
add how to sign icla，

Reply:
if  already sign icla， follow this：

1. provide request account template：

Prospective userid:  xxx
Full name: XXX
Forwarding email address（sign icla send eamil）: 
What time to receive the reply signed by icla ：